flink-clients/src/test/java/org/apache/flink/client/deployment/application/ApplicationDispatcherBootstrapTest.java

```diff
@@ -601,7 +601,7 @@ public void testDuplicateJobSubmissionWithTerminatedJobId() throws Throwable {
      * In this scenario, job result is no longer present in the {@link
      * org.apache.flink.runtime.dispatcher.Dispatcher dispatcher} (job has terminated and job
      * manager failed over), but we know that job has already terminated from {@link
-     * org.apache.flink.runtime.highavailability.RunningJobsRegistry running jobs registry}.
+     * org.apache.flink.runtime.highavailability.JobResultStore job result store}.
```

Comment:
```suggestion
     * org.apache.flink.runtime.highavailability.JobResultStore}.
```
nit: I prefer having the class name in the JavaDoc if possible. That means less work on renamings later on. I'm not going to mark each of these cases in this PR.